CHANGELOG.md

```diff
@@ -27,7 +27,20 @@ Dropping a requirement of a major version of a dependency is a new contract.
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.18.1...master
 
 ### Added
-- Add optional `IssueNavigatorView` parameter to JQL actions constructors. Make `IssueNavigationPage` support both list view and split view. Fix [JPERF-1043]. 
+- Support `ListView`, `DetailView` and other possible `IssueNavResultsView`s. Fix [JPERF-1043].
+- Add `SearchIssues` action, which supports various `IssueNavResultsView` and filters.
+- Add `IssueNavResultsView` parameter to JQL actions constructors.
+
+### Deprecated
+- Deprecate most methods of `IssueNavigatorPage`,
+  because they implicitly assumed `DetailView`, which won't work on `ListView`.
+- Deprecate all copies of `SearchJqlAction` (in favour of `SearchIssues`):
+  - `SearchJqlAction` itself
+  - `SearchJqlChangelogAction`
+  - `SearchJqlSimpleAction`
+  - `SearchJqlWildcardAction`
```

Comment:
Yes, `SearchIssues` is more code now, but it allows us to delete all that code in the next major release.